### PR TITLE
Enable option for json output for query benchmarker

### PR DESCRIPTION
**Pull request by codyshepherd**
These changes move all log messaging for influx query benchmarks to stderr, and add an option to print results as JSON to stdout.

### CODE REVIEW

File: bulk_query/query.go [hunk 1]
```diff
@@ -597,52 +608,71 @@ func (q *QueryBenchmarker) processStats(telemetrySink chan *report.Point) {
...
+		}
+	}
+	q.json["totalQueries"] = q.totalQueries
+	q.json["wallClockTime"] = q.wallTook.Seconds()
+	q.json["queryRate"] = float64(q.totalQueries) / float64(q.wallTook.Seconds())
+	q.json["workers"] = q.workers
+	q.json["batchSize"] = q.batchSize
+	if q.doJson {
+		for k, v := range q.json {
+			if _, err := json.Marshal(v); err != nil {
```
Comment (danxmoran) [inline bulk_query/query.go:667]: What values of `v` would we expect to fail `json.Marshal` here?

File: bulk_query/query.go [hunk 2]
```diff
@@ -347,7 +362,7 @@ loop:
 			if q.gradualWorkersIncrease && !q.isBurnIn {
 				if q.gradualWorkersMax <= 0 || q.workers <= q.gradualWorkersMax {
 					for i := 0; i < workersIncreaseStep; i++ {
-						//fmt.Printf("Adding worker %d\n", workers)
+						//log.Printf("Adding worker %d\n", workers)
```
Comment (danxmoran) [inline bulk_query/query.go:365]: Should we delete this comment instead of editing it?

File: bulk_query/query.go (hunk 1 above)
Comment (codyshepherd) [reply bulk_query/query.go:667]: Some of the stats can come up as `+Inf` or `-Inf`, which Marshal cannot handle.

File: bulk_query/query.go (hunk 2 above)
Comment (codyshepherd) [reply bulk_query/query.go:365]: oh yeah, probably. will do

**Review by danxmoran — APPROVED** [review]

**Review by mkm-influx — APPROVED** [review]

**Pull request merged**